setup/gcp/installer/forseti_installer.py

```diff
@@ -45,29 +45,47 @@ def __init__(self):
         pass
 
     def run_setup(self):
-        """Run the setup steps"""
+        """Run the setup steps."""
         utils.print_banner('Installing Forseti {} v{}'.format(
             self.config.installer_type, utils.get_forseti_version()))
 
-        # Preflight checks
+        # Preflight checks.
         self.preflight_checks()
 
-        # Deployment
-        bucket_name = self.generate_bucket_name()
-        conf_file_path = self.generate_forseti_conf()
-        deployment_tpl_path = self.generate_deployment_templates()
+        # Create/Reuse service account(s).
+        self.create_or_reuse_service_accts()
 
+        # Create configuration file and deployment template.
+        (conf_file_path,
+         deployment_tpl_path) = self.create_conf_file_dpl_template()
+
+        # Deployment.
+        bucket_name = self.generate_bucket_name()
         deploy_success, deployment_name = self.deploy(deployment_tpl_path,
                                                       conf_file_path,
                                                       bucket_name)
 
-        # After deployment
+        # After deployment.
         self.post_install_instructions(deploy_success,
                                        deployment_name,
                                        deployment_tpl_path,
                                        conf_file_path,
                                        bucket_name)
 
+    def create_conf_file_dpl_template(self):
```

Comment:
The reason why I am using abbreviations in some of the function names is because one of the pylint checks complains about function names that are longer than 30 characters, do you suggest disabling that pylint check in this case?

Reply:
Coming from java world, I don't want mind more verbose namings, even if longer than 30 characters.  But looking at this method in detail, perhaps it should not be so tightly tied to the 2 files it's actually producing.  So, how about something more generic like ```create_resource_files()```.  So that if this method produce more files in the future, this method naming can be more flexible?